src/adapter/src/coord.rs

```diff
@@ -1034,6 +1034,10 @@ impl Coordinator {
         self.catalog_transact(Some(&Session::dummy()), linked_cluster_ops)
             .await?;
 
+        // Signal to the storage controller that it is now freee to reconcile
+        // its state with what it has learned from the adapter.
+        self.controller.storage.reconcile_state().await;
```

Comment:
We're actively trying to improve bootstrap time. This function looks like it does some stash scans and I'm not sure what else. How fast is it? Is there a way to run this not blocking in the main coord startup thread?

Reply:
This will do a scan of a table that is empty in all cases except when recovering from a crash during boot. This could be moved to a background task if the scan of an empty table significantly impacts boot time.

Reply:
It doesn't significantly impact boot time. But in order to get sub 1s, we need to do probably 500 things that each improve time by 10ms, and this is one of them. I'm fine having it here to get the PR merged. But if it's possible to move to a background task, please do that in a followup.